packages/web3/package.json

```diff
@@ -60,6 +59,7 @@
     },
     "devDependencies": {
         "definitelytyped-header-parser": "^1.0.1",
-        "dtslint": "0.4.2"
+        "dtslint": "0.4.2",
+	"@types/node": "^12.6.1"
```

Comment:
The indentation here is wrong, can you make sure that you use spaces here and not tabs for consistency (please add this via rebase to the existing commit)?